airbyte-webapp/src/packages/cloud/services/auth/AuthService.tsx

```diff
@@ -65,6 +65,8 @@ interface AuthContextApi {
   logout: AuthLogout;
 }
 
+const _24_HOURS = 24 * 60 * 60 * 1000;
```

Comment:
I think we should just inline this instead of adding a `_` prefixed variable here.

Reply:
I extracted it to make it more descriptive, but can inline instead, no problem!